spec/models/client_application_spec.rb

```diff
@@ -0,0 +1,31 @@
+require File.dirname(__FILE__) + '/../spec_helper'
+RSpec.describe ClientApplication do
+  fixtures  :client_applications, :oauth_tokens
+  before(:each) do
+    @user = User.find_by_id(1) || create(:user)
```

Comment:
Mentioned via PM too: it's a bit weird that this code is trying to use User ID 1 by default and falling back to creating a user – you mentioned some of these tests being pulled from the upstream library, which might be why?